sentry-android-fragment/src/main/java/io/sentry/android/fragment/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -38,6 +55,8 @@ class SentryFragmentLifecycleCallbacks(
         savedInstanceState: Bundle?
     ) {
         addBreadcrumb(fragment, "created")
+
+        startTracing(fragment)
```

Comment:
Seems like we have a single instance of this class for the whole app regardless of Sentry.init or Hub.
We could have a publish subscriber apporach and have this class only 'raise events'. And a FragmentIntegration that upon `register` we subscribe to events from this class.

Reply:
It's an instance per activity, and it does the same approach as `ActivityLifecycleIntegration`, we could go with pub/sub yes, but I am not sure that it brings any benefit compared with the current approach, mind elaborating?

Reply:
Clarified on a call, makes sense